src/components/ui/IconButton/icon-button.scss

```diff
@@ -0,0 +1,31 @@
+@import "../../../styles/utilities";
+
+.icon-button[data-store-icon-button] {
```

Comment:
Do we need `.icon-button` or can we go just with `[data-store-icon-button]`?

Reply:
actually, we can go just with [data-store-icon-button].